src/build_info.cpp

```diff
@@ -521,6 +521,15 @@ inline std::string geometry_to_string(point p)
 	return std::to_string(p.x) + 'x' + std::to_string(p.y);
 }
 
+template<typename coordinateType>
+inline std::string geometry_to_string(coordinateType horizontal, coordinateType vertical)
+{
+	// Use a stream in order to control significant digits in non-integers
```

Comment:
The default according to cppreference is 6 digits of precision, which seems inappropriate. I think it would be better to set the desired precision in code, rather than commenting while leaving it at 6.

Reply:
Oops. Set the precision in the new version and also switched to using `formatter` since it's already in use in the file.